Octokit.Tests/Clients/OrganizationsClientTests.cs

```diff
@@ -130,7 +176,43 @@ public class TheGetAllForCurrentMethod
                 await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAllForCurrent(null));
             }
         }
+        
+        public class TheGetAllOrganizationsMethod
+        {
+            [Fact]
+            public async Task RequestsTheCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new OrganizationsClient(connection);
+
+                await client.GetAll();
 
+                connection.Received().GetAll<Organization>(Arg.Is<Uri>(u => u.ToString() == "organizations"));
+            }
+
+            [Fact]
+            public async Task RequestsTheCorrectUrlWithApiOptions()
```

Comment:
Since this isn't actually using `ApiOptions` it should be named something else, perhaps `RequestsTheCorrectUrlWithRequestParameter`